README.md

```diff
@@ -1,9 +1,101 @@
 <a href="https://www.lacework.com/"><img src="https://www.lacework.com/wp-content/uploads/2019/07/Lacework_Logo_color_2019.svg" width="250px" height="100px" title="Lacework" alt="Lacework"></a>
 
-# Lacework Agent Release
-The source for officially supported Lacework agent releases
+# Lacework Agent Releases
+This Lacework agent GitHub site provides the officially supported Lacework agent releases. This site provides both the Lacework agent install and source files available for downloading.
 
-# Lacework Agent Documentation
-Install and configure agent, use the polygraph, and learn about the host dossiers <a href="https://support.lacework.com/hc/en-us/categories/360001044834-Lacework-for-Workload-Security" target="_blank">**here**</a>.
+# Releases
+In the right frame, click **Releases** to view all the available agent releases.
+Under the **Assets**, all the release download files are listed. These download files contain the agent installers and the source code. In addition, each release contains a direct link to the specific agent release notes.
+For all the agent release notes, see [Release Updates](https://support.lacework.com/hc/en-us/categories/360000539793-Release-Updates).
+
+# Install Agents
+This README.md file provides installation instructions for installing the agent from this Lacework Agent GitHub repository.
+You can also install the agent from the Lacework Console. For a listing of the different agent install methods, see [Agent Install Options](https://support.lacework.com/hc/en-us/sections/360002375733).
+
+## Install Prerequisites
+Complete the following prerequisites steps before starting any of the agent install procedures below.
+1. Download the release package ```release.tgz``` file from this Lacework GitHub Agent Release repository, where ```release``` is replaced by the agent release number.
+    1. In the right frame, click **Releases** to view all the available agent releases.  
+    2. Find a release and click ```release.tgz``` to download the file that contains the agent installers, where ```release``` is replaced by the agent release number.
```

Comment:
Please add a section which describes validation of the the release tar images using the checksum.txt for the release. ```md5sum -c checksum.txt```

Reply:
Done